### PR TITLE
Revert "do not process stdderr when running df (#125)"

### PR DESCRIPTION
**Pull request by sxa**
This reverts commit 4cbafed37b95cfb809d7751ca0ab13624b9bd841 from https://github.com/adoptium/STF/pull/125 and implements it properly - something went wrong as I was rebasing

### CODE REVIEW

**Comment by sxa** [comment]
Testing on [Windows](https://ci.adoptopenjdk.net/view/Test_system/job/Test_openjdk11_j9_extended.system_x86-64_windows/587/console) [AIX](https://ci.adoptopenjdk.net/view/Test_system/job/Test_openjdk11_j9_extended.system_ppc64_aix/234/console) [Mac](https://ci.adoptopenjdk.net/view/Test_system/job/Test_openjdk11_j9_extended.system_x86-64_mac/511/console)

File: stf.core/scripts/stf.pl [hunk 1]
```diff
@@ -748,7 +749,7 @@ sub check_free_space {
 	}
 	else {
 		$cmd = "df -k $results_root";
-		my @df_output = `$cmd 2>&1`;
+		my @df_output = `$cmd`;
```
Comment (Mesbah-Alam) [inline stf.core/scripts/stf.pl:752]: This change wasn't part of https://github.com/adoptium/STF/pull/125.

**Comment by AdamBrousseau** [comment]
Looking good now
```
Retrieving amount of free space on drive containing F:/Users/jenkins/workspace/Test_openjdk8_j9_extended.system_x86-64_windows_Nightly_testList_2/aqa-tests/\TKG\output_16488267409147\DaaLoadTest_all_CS_5m_0
11:25:47  There is 92881 Mb free
```
https://openj9-jenkins.osuosl.org/job/Test_openjdk8_j9_extended.system_x86-64_windows_Nightly_testList_2/241/
Thanks @sxa

**Review by smlambert — APPROVED** [review]

File: stf.core/scripts/stf.pl (hunk 1 above)
Comment (sxa) [reply stf.core/scripts/stf.pl:752]: No - it's the bit that should have been in 125 to fix a problem we were seing on some of our mac systems :-)

**Pull request merged**